python/common/python/datetime.java

```diff
@@ -0,0 +1,36 @@
+package python;
+
+@org.python.Module(__doc__ = "The datetime module supplies classes for manipulating dates and times in both simple and complex ways.\n")
+public class datetime extends org.python.types.Module {
+    public datetime() {
+        super();
+    }
+
+    @org.python.Attribute
+    public static org.python.Object __file__ = new org.python.types.Str("python/common/python/datetime.java");
+    @org.python.Attribute
+    public static org.python.Object __loader__ = org.python.types.NoneType.NONE; // TODO
+    @org.python.Attribute
+    public static org.python.Object __name__ = new org.python.types.Str("datetime");
+    @org.python.Attribute
+    public static org.python.Object __package__ = new org.python.types.Str("");
+    @org.python.Attribute
+    public static org.python.Object __spec__ = org.python.types.NoneType.NONE; // TODO
+
+    @org.python.Method(__doc__ = "date() -> new datetime object", default_args = { "year", "month", "day" })
```

Comment:
Appropiate way would be to actually expose the constructor akin to how they have done it with OrderedDict (see https://github.com/Melkster/voc/blob/master/python/common/python/collections.java#L25-L51). 

What you have done here is simply a method which invokes the constructor for you, creating unneeded indirection